kubernetes/docs/change-graph-metadata.md

```diff
@@ -0,0 +1,57 @@
+Changing the Graph Id and Description
+=======================================
+By default, the default Gaffer deployment ships with the Graph name "simpleGraph" and description "A  graph for demo purposes" These are just placeholders and can be overwritten. This guide will show you how.
+
+The first thing you'll need to do is [deploy the demo graph](./deploy-demo-graph.md).
+
+### Changing the description
+Create a file called `graph-meta.yaml`. We will use this file to add our description and graph Id.
+Changing the description is as easy as changing the `graph.config.description` value.
+```yaml
+graph:
+  config:
+    description: "My graph description"
+```
+Feel free to be a bit more imaginative.
+
+### Deploy the new description
+Upgrade your deployment using helm:
+
+```bash
+helm upgrade my-graph gaffer-docker/gaffer -f graph-metadata.yaml --reuse-values
+```
+
+The `--reuse-values` argument means we don't override any passwords that we set in the initial construction.
+
+You can see your new description if you go to the Swagger UI and call the /graph/config/description endpoint.
+
+### Updating the graph Id
+Updating the graph Id is a little more complicated since the Graph Id corresponds to an Accumulo table. We have to graph the gaffer user permissions to read and write to that table. To do that update the `graph-meta.yaml` file with the following contents:
```

Comment:
```suggestion
Updating the graph Id is a little more complicated since the Graph Id corresponds to an Accumulo table. We have to change the gaffer user's permissions to read and write to that table. To do that update the `graph-meta.yaml` file with the following contents:
```